include/zephyr/bluetooth/classic/avrcp.h

```diff
@@ -24,6 +24,13 @@ struct bt_avrcp_unit_info_rsp {
 	uint32_t company_id;
 };
 
+struct bt_avrcp_subunit_info_rsp {
+	uint8_t subunit_type;
+	uint8_t max_subunit_id;
+	const uint8_t *extended_subunit_type; /**< contains max_subunit_id items */
```

Comment:
Incorrect comments

Reply:
Hi @lylezhu2012 
It says that the maximum_subunit_id = total number - 1,
whereas extended_subunit_type contains the subunits other than the basic one, which is also total number - 1.
![image](https://github.com/user-attachments/assets/3e0899b5-6605-40b7-9b38-0947c5f64200)

Reply:
For the most cases of AVRCP applications, max_subunit_ID = 0, and there are NO extended_subunit_type or extended_subunit_ID.
![image](https://github.com/user-attachments/assets/015aa575-260c-44fe-b8ab-4bcdf419776c)

Reply:
> I mean whether the comments should be `/**< contains max_subunit_type items */` instead of `/**< contains max_subunit_id items */`?

Hi @lylezhu2012 
I think extended_subunit_type do have max_subunit_id items.

For example, if **maximum_subunit_id = 4**, then there are totally 5 subunits, let's say subunit 0,1,2,3,4.

The first **subunit_type** refers to subunit 0.

The **extended_subunit_type** contains subunit [1,2,3,4].

The exact order of **extended_subunit_type** is recorded by **extended_subunit_id**

Typically, extended_subunit_id would be [1,2,3,4], however, it can be [1,4,2,3], or any other oders as long as nothing duplicated.

Reply:
> > Hi @lylezhu2012 It says that the maximum_subunit_id = total number - 1, whereas extended_subunit_type contains the subunits other than the basic one, which is also total number - 1. ![image](https://private-user-images.githubusercontent.com/174715478/390939987-3e0899b5-6605-40b7-9b38-0947c5f64200.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4NTkwOTksIm5iZiI6MTczMjg1ODc5OSwicGF0aCI6Ii8xNzQ3MTU0NzgvMzkwOTM5OTg3LTNlMDg5OWI1LTY2MDUtNDBiNy05YjM4LTA5NDdjNWY2NDIwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQwNTM5NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZjNiYjc1Yzg3ODRjZDliZjk5ODI5MTgxMjEyZTM0NzFhZjJkMGE1OGIwYTc5YmRiMDhlODUzNGU2YmIyMWM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hg7rdgVMBBtNXJUTHMpYAhmmlV4iLLxb6sKEvfGxOs0)
> 
> I have a question, if maximum_subunit_id = 4, whether it means the rsp contains extended_subunit_type[1:5]? Or the rsp contains extended_subunit_type[1:4]?

Hi @lylezhu2012 

If maximum_subunit_id = 4, then there are 5 subunits in this response. One in subunit_type, the others in extended_subunit_id.

Reply:
> > Hi @lylezhu2012 It says that the maximum_subunit_id = total number - 1, whereas extended_subunit_type contains the subunits other than the basic one, which is also total number - 1. ![image](https://private-user-images.githubusercontent.com/174715478/390939987-3e0899b5-6605-40b7-9b38-0947c5f64200.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4NTkwOTksIm5iZiI6MTczMjg1ODc5OSwicGF0aCI6Ii8xNzQ3MTU0NzgvMzkwOTM5OTg3LTNlMDg5OWI1LTY2MDUtNDBiNy05YjM4LTA5NDdjNWY2NDIwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQwNTM5NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZjNiYjc1Yzg3ODRjZDliZjk5ODI5MTgxMjEyZTM0NzFhZjJkMGE1OGIwYTc5YmRiMDhlODUzNGU2YmIyMWM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hg7rdgVMBBtNXJUTHMpYAhmmlV4iLLxb6sKEvfGxOs0)
> 
> I have a question, if maximum_subunit_id = 4, whether it means the rsp contains extended_subunit_type[1:5]? Or the rsp contains extended_subunit_type[1:4]?

In the most cases (if the Bluetooth device do not support AV/C related profile other than AVRCP), **max_subunit_id** would be 0, and there is totally 1 subunit. One in **subunit_type**, nothing in **extended_subunit_type** and **extended_subunit_id**.
All octets in **extended_subunit_type** and **extended_subunit_id** would be 0xFF in this case.